### ISSUE TITLE
Feature Request: Add GitHub Search / Theme / Monospaced Font

### ISSUE THREAD

**Issue by p-realinho** [bluecaret/carettab#19]
Hi!
First of all, thanks for this great extension.

Second, I think it'd be useful to have GitHub as a search provider too, since the extension kinda looks like it could be targeted for tech people (hence the brackets and all...? At least that's what it looks like to me.)

I've actually implemented this locally, so let me know if I should go ahead and submit a PR.

Another thing I thought would be cool, was to have a theme along the lines of "Monokai" or some other color scheme like that, and have the brackets be a different color from the rest (to sort of emulate syntax highlighting).
Also, maybe a monospaced font too (like Roboto Mono for example) ... 
Basically to make it look more like a code editor ^_^

Thanks!

**Comment by bluecaret** [comment]
Thank you for the suggestions! No need to put in a PR for that, I'll throw that in real quick on the next update since it really doesn't take long to do.

As far as additional fonts and color options, I'll be working on this for a future update and will definitely consider a monospaced font option. :)

**Comment by bluecaret** [comment]
Hello, I've just released an update that adds GitHub as a search option. Thank you.

**Comment by p-realinho** [comment]
Hi! Thank you for adding this feature.

I can't get it to work tho.
Whenever I select the "GitHub" option, it just behaves like the "Google" option.
It displays "Google" in the input placeholder and the search also goes to Google.

All the other search engines work fine.

**Comment by p-realinho** [comment]
Ops, I just noticed there's already an issue on this. #24

**Comment by bluecaret** [comment]
Thanks for the report. I hope to get to that this weekend. I swear I tested it first ;)

**Comment by p-realinho** [comment]
Seems to be fixed after the last update. Thanks! :)
